src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -473,6 +511,28 @@ function WorkspacesListPage() {
     const sortWorkspace = useCallback((workspaceItems: WorkspaceItem[]) => workspaceItems.sort((a, b) => localeCompare(a.title, b.title)), [localeCompare]);
     const [inputValue, setInputValue, filteredWorkspaces] = useSearchResults(workspaces, filterWorkspace, sortWorkspace);
 
+    const domains = useMemo(() => {
+        if (!allDomains) {
+            return [];
+        }
+
+        return Object.values(allDomains).reduce<DomainItem[]>((acc, domain) => {
```

Comment:
This variable name isn't very clear, maybe something like
```suggestion
        return Object.values(allDomains).reduce<DomainItem[]>((domainItems, domain) => {
```